README.md

```diff
@@ -133,15 +133,15 @@ npm i
 npm run certs:make
 
 # Build the SDK and watch for changes
-npm start
+npm run watch
 
 # Start the sample webserver
-npm run server:samples
+npm run start:samples
 
 # Follow the next section to trust certificate
 ```
 
-Access the samples at https://forgerock-sdk-samples.com:3000
+Access the samples at https://user.example.com:3000
```

Comment:
We need to move to a shared host as we start testing against more complex environments. Having a different host from the default installs of `forgeops` and other core products limits what you can test.